Cargo.toml

```diff
@@ -61,6 +61,7 @@ keywords = ["bitcoin", "lightning", "chaumian", "e-cash", "federated"]
 threshold_crypto = { version = "0.1", package = "fedimint-threshold-crypto" }
 tonic_lnd = { version = "0.1.3", package="fedimint-tonic-lnd", features = ["lightningrpc", "routerrpc"] }
 cln-rpc = { package = "fedimint-cln-rpc", version = "0.4.0" }
+fedimint-aead = { version = "0.0.1" }
```

Comment:
?

Reply:
By Mistake, I thought that I have to include this line in root Cargo.toml file as per the screenshot :)